### DISCUSSION TITLE
[Geo Search] Mass Select found tours ?

### DISCUSSION THREAD

**Discussion by mff73** [mytourbook/mytourbook#965] [Q&A] [answered]
Hello,
i use the geo search function, and when i arrives to a limited numbers of found tours, is it possible to select them physically (all) for example to add tags ?
Here i selected 33 tours
![image](https://user-images.githubusercontent.com/50132146/202140380-18008dda-5987-417b-b8b6-f8d484277730.png)

But right click doesn't give anything, and tour list is not updated with found tours ?

Thanks.

**Comment by wolfgang-ch** [comment, marked as answer]
Created a feature request https://github.com/mytourbook/mytourbook/issues/966

**Reply by wolfgang-ch** [reply]
A single tour is selected in `Tour Book` view when `Link with Other Views` is checked

![image](https://user-images.githubusercontent.com/1283445/202208140-f8419e45-fca9-4a6c-8a57-1950c3bc7415.png)
